src/components/Details.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable prefer-destructuring */
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import Header from './Header';
+import Navbar from './Navbar';
+import DetailsCard from './DetailsCard';
+import { getCountry } from '../redux/country/detailsSlice';
+
+function DetailScreen() {
+  const { state } = useLocation();
+  const dispatch = useDispatch();
+  const { country } = useSelector((state) => state.country);
+
+  useEffect(() => {
+    dispatch(getCountry(state.capital));
+  }, [dispatch, state.capital]);
+
+  let data = null;
+  if (country.length > 0) {
+    data = country[0];
+  }
+
+  return (
+    <div className="details-card" style={{ width: '100%' }}>
+      <Navbar country={data ? data.name.common : ''} />
+      <Header flag={data ? data.flags.png : ''} title={data ? data.name.common : 'independent'} population={data ? data.population : 0} />
+      <div className="detailsBanner">
+        Country info
+      </div>
+      <DetailsCard light title="Area" value={data ? data.area : ''} />
+      <DetailsCard title="Latitude" value={data ? data.latlng[0] : 0} />
+      <DetailsCard light title="Longitude" value={data ? data.latlng[1] : 0} />
+      <DetailsCard title="Time Zone" value={data ? data.timezones[0] : ''} />
+      <DetailsCard light title="Capital Latitude" value={data ? data.capitalInfo.latlng[0] : ''} />
+      <DetailsCard title="Capital Longitude" value={data ? data.capitalInfo.latlng[1] : ''} />
+    </div>
```

Comment:
- [ ] Note that to be aligned with style guidelines  kindly remove the white background on this page 

![image](https://user-images.githubusercontent.com/65068771/236547718-66a641a0-3e37-4ba8-8507-96162ce9e0f7.png)

Reply:
@andersonlebon  hope you are good 
dear if you take a look at my presentation video you will notice that there is no white space in the page, I assume you used responsive mode instead of a mobile device screen and you have increased the usual height of screen manually that is why is shows that white space.

Reply:
hey @MMhaidari I understand but even in your demo the white bg is present
<img width="472" alt="image" src="https://user-images.githubusercontent.com/65068771/236608282-3bd6fae7-7bcc-4603-a837-540e368dae0f.png">
Simply add a background to the body of your page I am sure it will solve it.